bundle/manifests/kueue-operator.clusterserviceversion.yaml

```diff
@@ -58,8 +58,8 @@ spec:
e9OT3OfQORZDDbueBHf336ZlcEHsxDb+3Hm4F381OYKKM2R3zhGJaGz0ydXRL/fDqHel0uHvMuZPIvcZ8LSZhgbjbtQ33WdnOl7Ij520vjLi1Peu1MHrVui0uaUNsQNkdrd49NdUTtJEd7j5dmd+IOYrBXmhjGK1P3yKnl+7jdYhT8fZ0ZfvZPPxTeAlU092Nez3L7xP1Nmp2uhXnO6aLn96pl8jh5snT6StzWV4AU3K+7fZJmRObm0/KHeJuuj05Y3HH4CCaZmA9cerEJisViuucoW/rnwkXf1egsTgisEdiEiInCNv3Pe0fn3G1hU4zxoFqLk5mX2LpRb7tEJMPmWBM9CtQDU7F1rJQ9m8i3pB3WhsI8fDOY1N5t8n++A499m0MZe7MM1bI3T3NUPuMm3JGg5gxTIeN9b7vUQxUSBK1KwSWjzvRG7B/z4JfmARKgIFwkrpn8lUrE25C+4IGDBhjigMWp0HhIpLRaLaR9gGnBSDrAzAkyqtnKdaNvybBLfQZpwMsaHn9gngPZGJW0pzyIFnwwOGKQc8UlSbMasyeciAZ0LzvBYS8r2CDe1mHwu2tAFfQW8Jd0xLOOPURgk2mvybFsDuKSYUsJG2JBEbvyUqylfedNhCpxDxBbcMzLJBWkMp88apiSHyjpF/LYiSXJXWEhgFQIPU/yCRjon2tCvgE4EtaYN7KpbIevt8228QtDS4To2QMMTDl2ksApTDbItucGmhljSNQRdrETQtWkRGwiWSf+wJt7WQPqLCrbyQiA+OT5gbQ3kRMQvu5PEYwtprK1JbZuU3AKsKUUM8baK2GGqXRNvbMkAt7EQB+Qbw7pEDTCWgY3mrwyN/g+p8VrCWtM2nKnB1mgps8Wi+cdk8pVU83innwCLuOHDbLYpnZeATEgpxE9EZzxK/0RSbgbn8BGn3/78OisBIabwBNuH7cUhE/NxeNEiGeAv+7rDaZL9clnhQk4h1iPO1m3lt5fhTTx+QTKL77d3QsyTM8l+dhYnphBralLXJ8bd65UlLw6tECF3hYudZR8SU5jGKNRi9Y8XF/E5gAoR665wmWfZW8RWN77GVIv1vrzEb+JxGIWreSh5KlxWWfYWMdtWWVOoxX5Z8tyAKGSVZD3uHp2r9OyQWIvfPe5Cu8/+iK9BVGHZjmEgWc2myyeTtfqyhUJixS3ODZSm3T/5yCOt0PKHIic0psPw3Vp92SIH/S+p5pYuX2snrr/4yiOt0FBXlzr8vPQrwLKz0P+Rai4zVailEg8d4+QKp5Coxz/prDzB46VvBU/r5Y6YLS1bClOJ+2lVVHHoK+/0PaNCVXq/3fjII1t/VUy0U6V5Ns288NGXs9YbjAq10dCH3tFJvATk+8MqXHG3ptDqvqlXlqHCp+Url7yrV9gSkMcbr7z5JgAkaTnDOTrhS0A++8qzlx5wQf7X3FxeDrL+ao6x6sIc49rCySWwRt+Gh6i/mmMOF93n3aEqcBcS6MPcN/D6qwVvN97RuYADDJ5ePR/uvONrPO4nzwLMHRr5PmRQz9vaNz9xU2fBw7SoTytcoPPNK/DVA9IzQHNmY14CApvIc4I/n6MQj/CX7fIyzRkQOgBlfInL3ocC3Bj5B2cJiPaL6KfjSWMTQkghacz1attTAgIXe3bjb0Wn05DcyWezq3nrr2DTXA4K+PPipKoHMuMP6/JisEkgBwGnO1Qio8h48K+/sofpIZZseXwND6lSOg2bb04B5iqX9PBdSKoOq4qv/EjAF86nA6queUKmroCv3kmVyDSBJyPgC+dZUnVYBl5hrA3diUHHjcnVYT3ji8ygq5IDzAyDyAWhAorMYgnQaxvKAWOU5NmHoGGqPRBrxofAU3I8QV91jbemsQRg4XXgvRRES+l0fJEZZPX8OPA8NUf0pw2oSoZz+8EXbxA+GoAvMrM70Wu0yZg4PfgqTaJHA6zvHKTQs4u6/dT6SKBJPfhmEYQItOEEX4IVs9OXy14zH28ubuIX+983UthwEyNP2sDhC5SmEmdlu/MKF4ub1z3NT2fDba8Et74LghyGZW0sr3i7qHCZSnzZaznX2HRzCsCB8cBO1GLo88XFKansc6a+6X4mGWKxiN9DaYxvkr31uGNLurzpch+YpGEG04n3733zz9ORusv2zShtvnB5z5tTcHR9PMZ97xSnz06E8dvvwfNo843ZIkSg28a9sNHaMf/RucxjWl2hoO1+bf0cH/x1jFGodILjuNW0BCRQ3jzGibhs+JOlo74a5sI2wCPxz3Njk0r8SlvbQh1X4+JKhLFcbxzKLlRCvhmhAG667QIXS57WmFuQR2wJiAtWEaqb3JfZZLhwb0YQPiTnoZNoJ64dGafXG4+8eQmIFySqvQb+saNR90kNezEkUmAjQ6WO62saLx55AXExmePr3ULRLdMoVpp1jpNDX+4JdfsNlmLLf/L5fztWFlQ5+1ztpi2apVo/q6iCLG1zDW0S9oysD6MWZvLhZdpPkdlMHyjbQpzNDg52bwot3OQDQIjken3gNwOcHPoS4aVEkIc7/AQCHB8LKfEwvShE1IM2hUP0IhfpExc6H9qh7cped2kSwBRJPNOFBx3eD3rJZMk/hLRCEo7hLZ1aiJ3djijZaJ598DIOtbnbHqRG/ozOApOFGKkS2QzMfhB/dm36YtdRPbx2kpfIPughckfwlpUbo7T385UrWHVwHCbGjVkn5P4R93QMPsKPQk/YXyNSpMidfAAdJtQ7DkH61OaRP+9cOFd3f0tP4piQb2JEymiwxesxzu6T1foRecBAys3wQcKFPJFjA1+kOTpGDZZXwroPVubk3+nh8QWj5nlSkDdE1RArK8ls41i9w0YyhUZW4RX/yCiSRIUTsqX8MTr3bciMOqW+mLTfIlMU2cb6X+D4JHdeHetH8losAQxzVOikm81GqdFspjsF3fwNvAKFQqFQKBQKhUKhUCgUCoVCoVAoFAqFQqFQKBTKNvwPfHcIWuueLxoAAAAASUVORK5CYII="
```

Comment:
Wouldn't this be purged on any new generation of the bundle?

Reply:
ah, good catch! Fixed it the subsequent push.